src/components/chart/_macro.spec.js

```diff
@@ -1170,7 +1200,7 @@ describe('Macro: Chart', () => {
                 );
 
                 test('THEN: it renders the description for accessibility', () => {
-                    expect($('.ons-u-vh').text()).toBe(accessibleDescription);
+                    expect($('#chart-audio-description-uuid').text()).toBe(accessibleDescription);
```

Comment:
Why is the id for this one "uuid" but the rest "invalid-chart-123" for example? Would be good to change this to match the others